A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/PFD/A320_Neo_PFD.html

```diff
@@ -5,7 +5,15 @@
     <div id="Mainframe">
         <div id="Electricity" state="off">
             <div id="HorizonContainer">
-                <jet-pfd-attitude-indicator id="Horizon" background="true"></jet-pfd-attitude-indicator>
+			<svg width="100%" height="100%" >
+				<defs>
+					<radialGradient id="Backlight" cx="50%" cy="50%" r="75%" >
+						<stop offset="0%" style="stop-color:rgb(23,30,45);stop-opacity:1" />						
+						<stop offset="100%" style="stop-color:rgb(23,30,45);stop-opacity:1" />
+					</radialGradient>					
+				</defs>
+			</svg>
```

Comment:
could you explain a little about whats going on here?

Reply:
> could you explain a little about whats going on here?

Thats is the svg image for the display background color.

Reply:
Sure i will reduce the backlight brightness. Also can you please post a screenshot of the glareshield issue?. I am unable to replicate it.

Reply:
Also, are you using the latest master version?. Because when i compared my changes with the latest version just before pushing the files. I have noticed all potentiometer index values are changed by aden to fix issue (fix s p o o k y brightness knob action) . Therefore i updated the indexes of all brightness knobs accordingly in my systems.cfg file.

Reply:
Glaresheild brightness stopped working for me also. Here is image ref of what we are referring to 
![image](https://user-images.githubusercontent.com/18389085/93836369-70637f80-fc7a-11ea-9c5a-5edeebb6b9a6.png)

Reply:
Oh ok let me take a look. Sorry guys, i was confused it with the flood lights. Will fix it!.

Reply:
Done!. Here are the screenshots. It is really hard to determine the angle of the light fall, since the tray is missing!. I would really appreciate any feedback from any real pilot here.

![Desktop Screenshot 2020 09 22 - 07 19 04 88](https://user-images.githubusercontent.com/53265961/93837935-9c482a80-fca5-11ea-9c0e-1b764e7a5b23.png)

![Desktop Screenshot 2020 09 22 - 07 17 47 58](https://user-images.githubusercontent.com/53265961/93837937-9e11ee00-fca5-11ea-876f-e8f93cc743ca.png)